legal-api/src/legal_api/services/involuntary_dissolution.py

```diff
@@ -141,6 +142,12 @@ def _get_businesses_eligible_query(exclude_in_dissolution=True):
                     _is_limited_restored(),
                     _is_xpro_from_nwpta()
                 )
+            ).\
+            order_by(
+                no_transition_filed_after_restoration.desc(),
+                _has_no_transition_filed_after_restoration().asc(),
```

Comment:
how do we handle the situation when there are multiple businesses have the same effective date? e.g. 
![image](https://github.com/bcgov/lear/assets/169101247/4d3f9927-58c7-4afe-b355-9056d0351804)

Let's say, we are going to select 100 businesses to put into Involuntary Dissolution process, and the 100th - 101st businesses have the same effective date. Then which one should we pick? Or this doesn't actually matter?

Other than these, LGTM

Reply:
Thank you for bringing this up, Eason. 

Right now, we don't have a specific requirement for this case, maybe the order between them doesn't matter.